opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/config/PollerConfigurationResource.java

```diff
@@ -59,6 +57,7 @@
 
     @GET
     @Produces({MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON, MediaType.APPLICATION_ATOM_XML})
+    // TODO: Patrick: do we still need this class?
```

Comment:
@fooker we should discuss this.

Reply:
Same here - just delete the whole class.

Reply:
+1 I removed it